.github/scripts/updateChangelog.js

```diff
@@ -6,12 +6,19 @@
  * @param {string} releaseDate the release date to be written to the changelog
  */
 function updateChangeLog(changelogString, version, releaseDate) {
-    const optionalBetaOrRc = '(-rc.d+)?(-(b|beta).d+)?';
-    const changelogVersionRegExp = new RegExp(
-      `\\[(development|develop|unreleased|${version})${optionalBetaOrRc}.*`,
-      'gi',
-    );
-    return changelogString.replace(changelogVersionRegExp, `[${version}] - ${releaseDate}`);
-  }
-  
-  module.exports.updateChangeLog = updateChangeLog;
+  const optionalBetaOrRc = '(-rc.d+)?(-(b|beta).d+)?';
+  const changelogVersionRegExp = new RegExp(
+    `\\[(development|develop|unreleased|${version})${optionalBetaOrRc}.*`,
+    'gi',
+  );
+
+  const lastReleaseVersion = changelogString.match(/\[(\d+.\d+.\d+)\]/)[1];
+  changelogString = changelogString.replace(changelogVersionRegExp, `[${version}] - ${releaseDate}`);
```

Comment:
Mutating input arguments is not recommended - let's use a new variable instead, please 🙏🏼